includes/variables.php

```diff
@@ -9,7 +9,10 @@
 $tsml_bounds = get_option('tsml_bounds');
 
 //get the secret cache location
-$tsml_cache = '/meetings.json';
+if (!$tsml_cache = get_option('tsml_cache')) {
```

Comment:
is this change intentional? or does this overwrite @brianw-area24 's recent change?

Reply:
First time I've seen it. I thought I was getting all the most recent changes from develop. Will revert...

Reply:
I think this is a side effect of when changes got pushed up to #596, and then reverted. With the reverts that @kiyote33 has done, I'm hoping we'll be able to merge everything into develop OK, but we'll want to check develop closely when we merge the rest of the PR's for this version.